README.md

```diff
@@ -209,7 +216,7 @@ and improve:
 ![Architecture](docs/img/modin_architecture.png)
 
 Visit the [Documentation](https://modin.readthedocs.io/en/latest/developer/architecture.html) for
-more information, and checkout [the difference between Modin and Dask!](https://github.com/modin-project/modin/tree/master/docs/modin_vs_dask.md)
+more information, and check out [the difference between Modin and Dask DataFrame!](https://github.com/modin-project/modin/tree/master/docs/modin_vs_dask.md).
```

Comment:
The link to Modin paper should be replaced to a newer one - https://people.eecs.berkeley.edu/~totemtang/paper/Modin.pdf. Under More information and Getting Involved section.